### DISCUSSION TITLE
Replace node with react custom components

### DISCUSSION THREAD

**Discussion by leonard-henriquez** [Milkdown/milkdown#926] [General]
I've seen in a issue that someone mention replacing a node with a custom react component (https://github.com/Milkdown/milkdown/issues/687)

Is it still a feature of Milkdown?
Is it documented somewhere?

Thanks in advance 🙏 🙂

**Comment by ylc395** [comment]
Maybe you can see some examples here: 

https://github.com/Milkdown/website/blob/ac8a56cbd847995e881cfa0d8d60ec8991e1d1bc/src/component/DocRenderer/HeadingWidget.tsx
